src/page-modules/assistant/types.ts

```diff
@@ -36,6 +41,10 @@ export const TripQuerySchema = z.object({
   searchMode: searchModeSchema.optional(),
   searchTime: z.number().optional(),
   cursor: z.string().optional(),
+  viaId: z.string().optional(),
+  viaLon: z.number().optional(),
+  viaLat: z.number().optional(),
+  viaLayer: z.union([z.literal('address'), z.literal('venue')]).optional(),
```

Comment:
This is another relevant example. We don't have a `ViaTripQuerySchema`, but the `TripQuery` **can** contain a via location. I think we should be consistent in the typings here.

Reply:
Is this handled, @jonasbrunvoll ?